content/en/servers/sprints/1/prep/index.md

```diff
@@ -0,0 +1,36 @@
++++
+title = 'prep'
+layout = 'prep'
+emoji= '📝'
+menu_level = ['sprint']
+weight = 1
+backlog= 'Module-Node'
```

Comment:
Probably one for follow up, but if we're happy with the Servers naming, then we should probably rename this too.